venvUtils/ensureVenv.py

```diff
@@ -94,7 +94,7 @@ def populate():
 			"-m",
 			"pip",
 			"install",
-			"setuptools==69.5.1",
+			"setuptools==72.0",
```

Comment:
```suggestion
			"setuptools~=72.0",
```

Reply:
As requested twice previously, please fixup the code comment above this line which references py2exe and setuptools

```
			# py2exe is not compatible with setuptools 70+
			# wheel must be manually installed when creating an non-isolated build with a custom setuptools version.
```

should be replaced with

```
			# wheel must be manually installed when creating an non-isolated build with a custom setuptools version.
```